docs/schema_mapping.py

```diff
@@ -254,20 +254,62 @@
         'Pydantic standard "format" extension',
         ''
     ],
+    [
+        'IPv4Address',
+        'string',
+        '{"format": "ipv4address"}',
+        'Pydantic standard "format" extension',
+        'IPv4 as used in ``ipaddress`` module',
+    ],
```

Comment:
This would be something like:

```Python
    [
        'IPv4Address',
        'string',
        '{"format": "ipv4"}',
        'JSON Schema Validation',
        '',
    ],
```